rendermodules/em_montage_qc/plots.py

```diff
@@ -220,7 +226,7 @@ def plot_defects(render, stack, out_html_dir, args):
 
     save(plot_tabs)
 
-    return out_html.name
+    return out_html
```

Comment:
same here... this has to return a filename not a file object.

Reply:
same comment as above